test/presets/bem.js

```diff
@@ -151,6 +151,43 @@ doAsync(function() {
 
 // Comments with url can have maximum line length > 120 ................... https://github.com/yandex/codestyle/blob/master/js.md
 
+// `i-bem` should be named BEM
+// `i-bem__dom` should be named BEMDOM
+modules.define('something-truly', ['i-bem__dom', 'something-else'], function(provide, BEMDOM, SomethingElse) {
+
+// Should use `this.name` if possible for BEMDOM.decl
+// Should have @lends jsdoc tag
```

Comment:
> I don't see why It is required.

It seems I was wrong. [`class` plugin for `jsd`][1] should work properly.

[1]: https://github.com/bem/jsd-plugins-bem/blob/08080237503b31b2f701110dcc4707071ead43d0/plugins/class.js